recipes/robustness/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "robustness" %}
+{% set version = "1.2.post1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 82b1cae2a5f16846fd4caadab9cd43fda14f752feec76cc79c38be185255ce99
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [not linux]
```

Comment:
I have the same question about `noarch: python` here is well. I think the recipe should be `noarch`, and the `skip` section can be removed.

Reply:
@npavlovikj this package doesn't work on osx or win. i've added noarch and removed the skip, but added a comment line to clarify that functionality is not available on those platforms.

Reply:
That is ok @oblute , thanks.